app/scraper/loot/scraper.py

```diff
@@ -36,10 +42,34 @@ def get_source() -> Source:
     def get_duration() -> OfferDuration:
         raise NotImplementedError("Please implement this method")
 
+    @staticmethod
+    def read_offers_from_page(driver: WebDriver) -> list[Offer]:
+        raise NotImplementedError("Please implement this method")
+
     @staticmethod
     def get_max_wait_seconds() -> int:
         return MAX_WAIT_SECONDS
 
+    @staticmethod
+    def categorize_offers(offers: list[Offer]) -> list[Offer]:
+        for offer in offers:
+
+            if re.search("(demo|trial)", offer.title[-6:], re.IGNORECASE):
+                offer.category = Category.DEMO
+                continue
+
+            if offer.duration == OfferDuration.ALWAYS:
+                offer.category = Category.ALWAYS_FREE
+                continue
+
+            if offer.valid_to is not None and offer.valid_to > datetime.now().replace(
+                tzinfo=timezone.utc
+            ) + timedelta(days=3650):

```

Comment:
Please add a comment as to why we need to look for 10 years in the future. Arguably this could be shortened to sth. like 1 year.

Also maybe the "ALWAYS_FREE" could be more like a "FREE_FOR_A_VERY_VERY_LONG_TIME" (yeah we need a better name ;-))

Reply:
10 years is forever for all intents and purposes.

Reply:
1 year could still be a serious offer, e.g. valve could make their games free for a year for their 30 year anniversary.
I don't think it matters if it's 2 or 10 years.

Reply:
I'm fine with 10 years. Let's leave it at that until someone finds out that offers are still displayed when they set the end date to now+9 years in Humble Store ;-)